### ISSUE TITLE
Can't seem to play remote Video

### ISSUE THREAD

**Issue by CeoFred** [peers/peerjs#682]
Peerjs handles connection, signalling and every other thing perfectly. I get my on stream event fired, and a mediaStreamObject is received from the remote end, but after i set the srcObject of a video element the video does not play, any reason why??

**Comment by Airell98** [comment]
teuing bray, aing ge kitu saruana

**Comment by bennlich** [comment]
@CeoFred does your video tag have an `autoplay` attribute? And are you seeing the same behavior in firefox and chrome?

**Comment by CeoFred** [comment]
Yes , I actually have solved that issues.. but I get another issue though,
on my live server , I seem to see a duplicate of peers videos.

**Comment by eysteinbye** [comment]
> Yes , I actually have solved that issues.. but I get another issue though, on my live server , I seem to see a duplicate of peers videos.
> […](#)

@CeoFred Can you share the solution to the problem? Why was your remote video playing?

**Comment by CeoFred** [comment]
So you actually need to have your internet connection up and running for
signaling to happen.. note that, your servers do little or no work with the
data transfer, only sets up a network connection between two or more
endpoints, I'll call the server a signalling server.. this is actually what
worked for me.

**Comment by iamafasha** [comment]
I have the same issue. Mine works perfectly for some devices and others it doesn't.

**Comment by iamafasha** [comment]
I am facing the same problem I tried following @CeoFred but I don't get it.  It looks like it works sometimes and sometimes it never works.

**Comment by CeoFred** [comment]
Well, you are not doing something right then
